Docs 6939 tokenization january2020 (#44)
mulesoft/docs-anypoint-security: modules/ROOT/pages/grant-tokenization-permissions.adoc

= Grant Tokenization Permissions
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> An administrator for your organization can grant these permissions to any user from Access Management.

Reviewer: ```suggestion
An administrator for your organization can grant management permissions to any user by using access management.
```
Per PMM list, lowercase "access management."

Author: Boo!

== Tokenization Permissions

> A user needs to have a set of predefined permissions to create and manage tokenization formats and services:

Reviewer: ```suggestion
A user must have a set of predefined permissions to create and manage tokenization formats and services:
```
internationalization

[%header%autowidth.spread,cols="a,a"]
|===
| Name | Description
| Manage Tokenization Formats
| Provides the ability to create, modify, and delete tokenization formats to use with the tokenization service in a specific environment.

| Manage Tokenization Services
| Provides the ability to create and deploy a tokenization service, as well as modify and delete a tokenization service in a specific environment.
|===

== Grant a User Tokenization Permissions

> 

Reviewer: Any transition text?
Should we mention the prereq of being an organization admin?

> . In Access Management, from the left navigation menu, click *Users*.

Reviewer: As above, PMMs say that this is not a product, so...is this the title of a page? Can we say "In the *Access Management* <something>, go to the left navigation menu and click *Users*."?

> . To grant permissions click the name of the user. 

Reviewer: ```suggestion
. To grant permissions to a user, click the user's name. 
```
OR
```suggestion
. Click the name of the user to whom you want to grant permissions. 
```

. In the *Permissions* tab, click *Tokenization*. 

> . Select *Manage Tokenization Formats* from the Permissions drop-down, and click the blue plus icon image:blue-plus-security.png[]. 

Reviewer: See my comment earlier about bold for menu names....
```suggestion
. Select *Manage Tokenization Formats* from the *Permissions* menu and click the blue "plus" icon image:blue-plus-security.png[]. 
```
Yay for screen-cap icons! I was just discussing this with Hanna. We need to advertise and promote this ability.

> . Select the environment from the Environment drop-down, then select the *Manage Tokenization Services* permission from the Permissions drop-down, and click the blue plus icon image:blue-plus-security.png[].

Reviewer: ditto
```suggestion
. Select the environment from the *Environment* menu, select the *Manage Tokenization Services* permission from the *Permissions* menu, and click the blue "plus" icon image:blue-plus-security.png[].
```
